tests/run-tests.sh

```diff
@@ -2,7 +2,107 @@
 
 set -e
 
+export QONTRACT_SERVER_NAME=qontract-server
+export QONTRACT_SERVER_IMAGE=quay.io/app-sre/qontract-server
+export QONTRACT_SERVER_IMAGE_TAG=64b433b
+
+export KEYCLOAK_NAME=keycloak
+export KEYCLOAK_IMAGE=quay.io/keycloak/keycloak
+export KEYCLOAK_IMAGE_TAG=17.0
+export KEYCLOAK_USER=admin
+export KEYCLOAK_PASSWORD=admin
+
+export KEYCLOAK_CLI_NAME=keycloak_cli
+export KEYCLOAK_CLI_IMAGE=quay.io/tpate/keycloak-config-cli
+export KEYCLOAK_CLI_TAG=latest
+
+export VAULT_NAME=vault-dev-server
+export VAULT_IMAGE=quay.io/app-sre/vault
+export VAULT_IMAGE_TAG=1.5.4
+export VAULT_ADDR=http://127.0.0.1:8200
+export VAULT_TOKEN=root
+export VAULT_AUTHTYPE=token
+
+cleanup () {
+  echo "cleaning"
+  if docker ps -a --format "table {{.Names}}" | grep -qw $KEYCLOAK_CLI_NAME; then
+    docker rm -f $KEYCLOAK_CLI_NAME
+  fi
+  if docker ps -a --format "table {{.Names}}" | grep -qw $KEYCLOAK_NAME; then
+    docker rm -f $KEYCLOAK_NAME
+  fi
+  if docker ps -a --format "table {{.Names}}" | grep -qw $QONTRACT_SERVER_NAME; then
+    docker rm -f $QONTRACT_SERVER_NAME
+  fi
+  if docker ps -a --format "table {{.Names}}" | grep -qw $VAULT_NAME; then
+    docker rm -f $VAULT_NAME
+  fi
+  echo "Docker environment cleaned"
+}
+
+container_alive () {
+  idx=0
+  until $(curl --output /dev/null --silent --head --fail $1); do
+    printf '.'
+    sleep 1
+    if [[ $idx == $2 ]]; then
+      cleanup
+      exit 1
+    fi
+    ((++idx))
+  done
+}
```

Comment:
add lines between done and }

echo ""
echo "connectivty established with $3"
echo ""

container_alive xxx $CONTAINER_HEALTH_TIMEOUT_DEFAULT $QONTRACT_SERVER_NAME

or whatever name is being run

Reply:
also for completeness do the beginning too